tests/Rector/ClassMethod/MigrateAtToConsecutiveExpectationsRector/Fixture/skip_missing_expects.php.inc

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Rector\PHPUnit\Tests\Rector\ClassMethod\MigrateAtToConsecutiveExpectationsRector\Fixture;
+
+class Foo
+{
+    public function someMethod()
+    {
+    }
+}
+final class SkipMissingExpects extends \PHPUnit\Framework\TestCase
+{
+    protected $mock;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->mock = $this->createMock(Foo::class);
+    }
+
+    public function test(): void
+    {
+        $this->mock
+            ->method('someMethod')
+            ->willReturn(1);
+
+        $mock = $this->createMock(Foo::class);
+
+        $mock
+            ->method('someOtherMethod')
+            ->willReturn(1);
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\PHPUnit\Tests\Rector\ClassMethod\MigrateAtToConsecutiveExpectationsRector\Fixture;
+
+class Foo
+{
+    public function someMethod()
+    {
+    }
+}
+final class SkipMissingExpects extends \PHPUnit\Framework\TestCase
+{
+    protected $mock;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->mock = $this->createMock(Foo::class);
+    }
+
+    public function test(): void
+    {
+        $this->mock
+            ->method('someMethod')
+            ->willReturn(1);
+
+        $mock = $this->createMock(Foo::class);
+
+        $mock
+            ->method('someOtherMethod')
+            ->willReturn(1);
+    }
+}
+
+?>
```

Comment:
Hi, could you simplify test to fewer lines? The diff of changes should be more obvious

Reply:
Sure - how's it look now?

I've removed the comparison as it should be skipped (and not refactored).

Reply:
Better, thanks :+1: 

What happens with this code without the change?

Reply:
It will throw an error.

If the `expects()` is missing, the `var` property of `$methodCall` (line 151) will be a `PropertyFetch` or `Variable` (depending where the object is stored) rather than the expected `MethodCall` object (for `expects()`).

Error would be threw at line 67 - `isValidExpectsCall()` which expects a `MethodCall` object as a parameter.

Reply:
I see, thanks :+1: